src/Form/PretixConfigForm.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Drupal\itk_pretix\Form;
+
+use Drupal\Core\Form\ConfigFormBase;
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Class PretixConfigForm.
+ */
+class PretixConfigForm extends ConfigFormBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function getEditableConfigNames() {
+    return [
+      'itk_pretix.pretixconfig',
+    ];
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId() {
+    return 'pretix_config_form';
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateInterface $form_state) {
+    $config = $this->config('itk_pretix.pretixconfig');
+    $form['api_token'] = [
+      '#type' => 'textfield',
+      '#title' => $this->t('API Token'),
+      '#maxlength' => 64,
+      '#size' => 64,
+      '#default_value' => $config->get('api_token'),
+    ];
+    return parent::buildForm($form, $form_state);
```

Comment:
We need some more settings here, e.g. `pretix url` and `organizer (short form)`.

Reply:
I wasn't quite sure what values you needed so i left that part for you